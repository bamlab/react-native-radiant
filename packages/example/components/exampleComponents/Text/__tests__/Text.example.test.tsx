import React from 'react';
import { render } from '@testing-library/react-native';
import TextExample from '../Text.example';
import { renderComponentToWebScreenshot } from 'kin';

describe('TextExample', () => {
  it('should render the TextExample component correctly', () => {
    const { toJSON } = render(<TextExample />);
    expect(toJSON()).toMatchSnapshot();
  });

  it('should match image snapshot', async () => {
    const renderedComponent = render(<TextExample />);

    expect(await renderComponentToWebScreenshot({ renderedComponent })).toMatchImageSnapshot();
  });
});
