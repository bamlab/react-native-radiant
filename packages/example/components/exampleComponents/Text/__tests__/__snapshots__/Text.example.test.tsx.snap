// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextExample should render the TextExample component correctly 1`] = `
<Text>
  This is a text
</Text>
`;
